backend/battle/forms.py

```diff
@@ -2,17 +2,22 @@
 from django.contrib.auth.forms import UserCreationForm
 
 from battle.models import Battle, PokemonTeam, Team
-from battle.battles.battle import validate_sum_pokemons, verify_pokemon_is_saved
+from battle.battles.battle import save_pokemons_if_they_doenst_exist
```

Comment:
This namespace is weird: `battle.battles.battle`. I'm not gonna stress about this but we should avoid this going forward

Reply:
okay, thanks for alert me